chart/validator/values.yaml

```diff
@@ -359,6 +359,82 @@ plugins:
         protocol: TCP
         targetPort: https
       type: ClusterIP
+- chart:
+    name: validator-plugin-oci
+    repository: "https://spectrocloud-labs.github.io/validator-plugin-oci"
+    version: v0.0.1
+  values: |-
+    controllerManager:
+      kubeRbacProxy:
+        args:
+        - --secure-listen-address=0.0.0.0:8443
+        - --upstream=http://127.0.0.1:8080/
+        - --logtostderr=true
+        - --v=0
+        containerSecurityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+        image:
+          repository: gcr.io/kubebuilder/kube-rbac-proxy
+          tag: v0.15.0
+        resources:
+          limits:
+            cpu: 500m
+            memory: 128Mi
+          requests:
+            cpu: 5m
+            memory: 64Mi
+      manager:
+        args:
+        - --health-probe-bind-address=:8081
+        - --leader-elect
+        containerSecurityContext:
+          allowPrivilegeEscalation: false
+          capabilities:
+            drop:
+            - ALL
+        image:
+          repository: quay.io/spectrocloud-labs/validator-plugin-oci
+          tag: v0.0.1  # x-release-please-version
+        resources:
+          limits:
+            cpu: 500m
+            memory: 128Mi
+          requests:
+            cpu: 10m
+            memory: 64Mi
+      replicas: 1
+      serviceAccount:
+        annotations: {}
+    kubernetesClusterDomain: cluster.local
+    metricsService:
+      ports:
+      - name: https
+        port: 8443
+        protocol: TCP
+        targetPort: https
+      type: ClusterIP
+
+    # Optional environment variable configuration
+    env: []
+    # - name: HTTP_PROXY
+    #   value: http://192.168.1.100:8080
+    # - name: HTTPS_PROXY
+    #   value: http://192.168.1.100:8080
+    # - name: NO_PROXY
+    #   value: foo.bar.com,127.0.0.1
+
+    # Optional proxy configuration. If enabled, the secret containing your proxy CA certificate
+    # will be mounted in the manager container and configured via an init container.
+    proxy:
+      enabled: false
+      # The image used by the init container. Must include the update-ca-certificates command.
+      image: quay.io/spectrocloud-labs/validator-certs-init:latest
+      # The name of a secret containing a proxy CA certificate.
+      # WARNING: the key whose value is the CA certificate must end with '.crt'.
+      secretName: proxy-cert
```

Comment:
i noticed the equivalent of this section was not included for the network plugin. was that because this base validator can now support handling the proxy configurations?

Reply:
yessir